CMakeLists.txt

```diff
@@ -227,8 +222,7 @@ endif()
 if(CELERITAS_USE_VecGeom)
   find_package(VecGeom 1.1.17 REQUIRED)
 
-  if((CELERITAS_USE_CUDA AND NOT VecGeom_CUDA_FOUND)
-      OR (NOT CELERITAS_USE_CUDA AND VecGeom_CUDA_FOUND))
+  if((CELERITAS_USE_CUDA AND NOT VecGeom_CUDA_FOUND))
```

Comment:
Can you confirm that the build works if vecgeom is cuda-enabled and celeritas isn't?

Reply:
Yes, it builds and passes all enabled tests locally.

100% tests passed, 0 tests failed out of 135

The following tests did not run:
	127 - celeritas/track/TrackInit (Disabled)
	135 - app/demo-loop (Disabled)
	138 - app/geo-check (Disabled)